components/__design__/horizontal-logos.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import Image from 'next/image';
+import type { Meta as MetaObj, StoryObj } from '@storybook/react';
+
+export const HorizontalLogos: StoryObj = {};
+
+export default {
+  title: 'Design System/Logos',
+  component: () => {
+    return (
+      <div>
+        <div>
+          <Image
+              src={`/static/images/logos/horizontal-light.svg`}
```

Comment:
Adding black background allow us to seed the logo 😁
```suggestion
              src={`/static/images/logos/horizontal-light.svg`}
               className="bg-black"
```